WordPress/gutenberg
# Simplify zoom out insertion UX

## What?


Simplify the UX of zoom out inserting:

- zoom out on automatically opens the inserter in the pattern library view
- zoom out off automatically closes the inserter
- there are no more [+] buttons
- the selected section expands space top and bottom
- unselecting closes the space
- by default insertion happens after current selection

## Why?


The [+] buttons are obtrusive and inconvenient as an affordance. The separator that they create is an orphan in the UI - there is no relation between the inserter and the space created. Even with the prompt in #65392 it's still an awkward interaction that one clicks a button to get some text.

Opening the inserter in the pattern library as soon as entering zoom out is also more in tune with the original intent of this view to be useful for page assembly.

## How?


Refactor some of the zoom out implementation. Remove the special inserters.

## Testing Instructions





- open zoom out
- **inserter should be open in pattern library**
- select a section
- **separators should appear both top and bottom**

### Testing Instructions for Keyboard


## Screenshots or screencast 

https://github.com/user-attachments/assets/4b79f1b1-364d-4dab-92f2-f523f2e963ad


**To review the intereaction. Is it worth pursuing this path?**
